bbc/simorgh
# WSTEAM1-995 UGC Uploader CSS and HTML semantics outside form

Resolves JIRA [995](https://jira.dev.bbc.co.uk/browse/WSTEAM1-995)

Overall changes
======
Updates HTML and CSS inline with styles and a11y criteria for outside the form for stage 1 

Code changes
======

- Sets page title to name of the form 
- Adds H2 subtitle from sectionText 
- Adds background gradient
- Adds background colour for form content
- Adds a css only divider line 
- Updates form size to be correct and different breakpoints


Testing
======
Check it on chrome, safari and firefox

1. Visit http://localhost.bbc.com:7081/mundo/send/u50853489?renderer_env=live and check it against the [designs for stage 1
](https://www.figma.com/file/hbI3T5RehCO1EUsb1p9aI3/Uploader-update---MVP---Handoff-file?type=design&node-id=2003-17388&mode=design&t=4FyI47yZxaAYDv7l-0)
2. Check it against the [a11y criteria ](https://paper.dropbox.com/doc/Uploader-Accessibility-acceptance-criteria--COZKDhVU3a7ON~pynADiTuoZAg-BN1tII16uRaecL0YQQ0pR)for things outside of the form, stage 1

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Designs ](https://www.figma.com/file/hbI3T5RehCO1EUsb1p9aI3/Uploader-update---MVP---Handoff-file?type=design&node-id=2003-17388&mode=design&t=4FyI47yZxaAYDv7l-0)

[a11y criteria](https://paper.dropbox.com/doc/Uploader-Accessibility-acceptance-criteria--COZKDhVU3a7ON~pynADiTuoZAg-BN1tII16uRaecL0YQQ0pR)

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
